dvc/commands/experiments/rename.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import logging
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link
+from dvc.exceptions import InvalidArgumentError
+from dvc.ui import ui
+
+logger = logging.getLogger(__name__)
+
+
+class CmdExperimentsRename(CmdBase):
+    def check_arguments(self):
+        if not bool(self.args.name) | bool(self.args.experiment):
+            raise InvalidArgumentError(
+                "Please provide an `experiment` and `name` argument."
+            )
+
+    def run(self):
+        from dvc.utils import humanize
+
+        self.check_arguments()
```

Comment:
This should be handled by argparse , I think the code would never run as it would fail when parsing args.

```suggestion
    def run(self):
        from dvc.utils import humanize
```